Features/Stories/FieldTypes/TextlineFieldType.feature

```diff
@@ -0,0 +1,247 @@
+        And I click at "OK" button
+        Then Publishing fails with validation error message "Form did not validate. Please review errors below."
+
+    @javascript @edge @broken
+    Scenario: Creating a valid content type with a text line Field works when using a minimum equal to the maximum character limit
+        Given I am on the "Content types" page
+        And I click in the "Content" Content type group
+        When I click at "Create a content type" button
+        And I fill form with:
+            | Field      | Value |
+            | Name       | Test  |
+            | Identifier | test  |
+        And I add a field type "Text line" with:
+            | Field           | Value |
+            | Name            | Text  |
+            | Identifier      | text  |
+            | Minimum length  | 4     |
+            | Maximum length  | 4     |
+        And I click at "OK" button
+        Then the Content is successfully published with the message "The Content type draft was successfully updated and published. Related Content has also been updated."
+
+    @javascript @edge @broken
+    Scenario: Creating an invalid content type with a text line Field fails when using a negative minimum character limit
+        Given I am on the "Content types" page
+        And I click in the "Content" Content type group
+        When I click at "Create a content type" button
+        And I fill form with:
+            | Field      | Value |
+            | Name       | Test  |
+            | Identifier | test  |
+        And I add a field type "Text line" with:
+            | Field           | Value |
+            | Name            | Text  |
+            | Identifier      | text  |
+            | Minimum length  | -1    |
+            | Maximum length  | 4     |
+        And I click at "OK" button
+        Then Publishing fails with validation error message "Form did not validate. Please review errors below."
+
+    @javascript @edge @broken
+    Scenario: Creating an invalid content type with a text line Field fails when using a maximum character limit equal to zero
+        Given I am on the "Content types" page
+        And I click in the "Content" Content type group
+        When I click at "Create a content type" button
+        And I fill form with:
+            | Field      | Value |
+            | Name       | Test  |
+            | Identifier | test  |
+        And I add a field type "Text line" with:
+            | Field           | Value |
+            | Name            | Text  |
+            | Identifier      | text  |
+            | Minimum length  | 0     |
+            | Maximum length  | 0     |
+        And I click at "OK" button
+        Then Publishing fails with validation error message "Form did not validate. Please review errors below."
+
+    @javascript @edge @broken
+    Scenario: Creating an invalid content type with a text line Field fails when using a negative maximum character limit
```

Comment:
Overall, I'd separate ContentType from Content (edit or form rendering or whatever). Not sure yet if this would then belong in RepositoryForms or in PlatformUI.